docs/pages/versions/unversioned/sdk/image.mdx

```diff
@@ -5,5 +5,71 @@ packageName: 'expo-image'
 ---
 
 import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+
+**`expo-image`** is a cross-platform React component that loads and renders images.
+
+Main features:
+
+- Designed for speed
+- Support for many image formats (including animated ones)
+- Disk and memory caching
+- Supports [blurhash](https://blurha.sh), a compact representation of a placeholder for an image
+- Transitioning between images when the source changes (no more flickering!)
+- Implements the CSS [`object-fit`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) and [`object-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) properties (see [`contentFit`](#contentfit) and [`contentPosition`](#contentposition) props)
+- Uses performant [`SDWebImage`](https://github.com/SDWebImage/SDWebImage) and [`Glide`](https://github.com/bumptech/glide) under the hood
+
+<PlatformsSection android emulator ios simulator web />
+
+## Installation
+
+> **Info** `expo-image` is not yet available in Expo Go and Snack.
```

Comment:
The callout here does provide where it doesn't work, however, I think it will be good to add a link to Development builds or re-phrase it like below so that it does not create confusion about the usability and provides a current way to use this library.

Re-phrasing something like below:

```
> **Info** `expo-image` is currently available with [Development builds](/development/create-development-builds/) and not in Expo Go and Snack yet.
```

Reply:
Good point 👍 What would you say about:

```mdx
> **Info** Currently `expo-image` can be used only with [development builds](/development/create-development-builds/) and bare React Native apps with [configured Expo modules](/bare/installing-expo-modules/).
> It is not available in Expo Go and Snack yet.
```

Reply:
Looks good 👍